at-hoangtran/Week3/Day4/Exercise2-Day4.rb

```diff
@@ -0,0 +1,9 @@
+def sum_arr(arr)
+  check = arr.all? { |x| x.is_a? Integer }
+  if check == true
+    puts sum_arr = arr.sum
+  else
+    puts nil
+ end
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/3cf28bfecc132fb6fcec164677df66b52ed8042f/at-hoangtran/Week3/Day4/Exercise2-Day4.rb#L7"></span>Layout/EndAlignment: <code>end</code> at 7, 1 is not aligned with <code>if</code> at 3, 2.</td>
    </tr>
  </tbody>
</table>